beater/gpfs.go

```diff
@@ -37,13 +37,15 @@ func (bt *Gpfsbeat) MmRepQuota() ([]QuotaInfo, error) {
 	var quotas []QuotaInfo
 	for _, filesystem := range bt.config.Filesystem {
 
-		cmd := exec.Command(bt.config.MMRepQuotaCommand, filesystem) // TODO: pass arguments
+		logp.Info("Running mmrepquota for filesystem %s", filesystem)
+
+		cmd := exec.Command(bt.config.MMRepQuotaCommand, "-Y", filesystem) // TODO: pass arguments
```

Comment:
There is CommandContext, which would allow that. Which timeout period did you have in mind?